addon/templates/components/uni-header-dropdown-item-simple.hbs

```diff
@@ -1,3 +1,4 @@
-<li class="uni-header__nav__dropdown__list__item">
-  <div {{action onClick option.key}}>{{option.value}}</div>
+<li class="uni-header__nav__dropdown__list__item" {{action onClick option.key}}>
+  <div class="uni-header__nav__dropdown__list__item--value">{{option.value}}</div>
+  <div class="uni-header__nav__dropdown__list__item--description">{{option.description}}</div>
```

Comment:
Be careful, you should check if we're sending description before adding this div, pretty sure this will break something